src/main/resources/org/jenkinsci/plugins/stashNotifier/StashNotifier/config.jelly

```diff
@@ -5,8 +5,14 @@
     </f:entry>
     <f:entry title="${%Credentials}" field="credentialsId">
       <c:select/>
-     </f:entry>
-     <f:entry title="Commit SHA-1" field="commitSha1">
+    </f:entry>
+    <f:entry title="Commit SHA-1" field="commitSha1">
+      <f:textbox />
+    </f:entry>
+    <f:entry title="Build Name" field="buildName">
```

Comment:
Why is this field being added in this PR?

Reply:
Because these were forgotten previously and might be also connected to the bug. And even if not, they are fixing the implementation for the fields of the initial issue.